src/App.jsx

```diff
@@ -1,17 +1,89 @@
-import { useState } from 'react'
-import mentorLogo from '/mentor.png'
-import './App.css'
+import {
+  BrowserRouter, Routes, Route, Navigate,
+} from 'react-router-dom';
+import { ReactDOM } from 'react';
+import PropTypes from 'prop-types';
+import './App.css';
+import { useSelector } from 'react-redux';
+import Login from './pages/Login';
+import Mentors from './pages/Mentors';
+import Registration from './pages/Registration';
+import AddMentor from './pages/AddMentor';
+import Welcome from './pages/Welcome';
+import Navbar from './components/Navbar';
+import MyReservations from './pages/MyReservations';
+import AddExpertise from './pages/AddExpertise';
+import DetailMentor from './pages/DetailMentor';
+
+function ProtectRoute({ isLoggedIn, children }) {
+  if (!isLoggedIn) {
+    return <Navigate to="/login" replace />;
+  }
+  return children;
+}
+
+ProtectRoute.propTypes = {
+  isLoggedIn: PropTypes.bool.isRequired,
+  children: PropTypes.instanceOf(ReactDOM).isRequired,
+};
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/hienphan0111/Mentor-front-end-app/assets/57421823/fcf5d4ed-b8d2-4e79-b27e-c92b04745ac8)

That way, your code will meet the latest standards. 💪